backend/src/models/reports/rpt_PA_Registered.js

```diff
@@ -0,0 +1,61 @@
+const dbConnection = require("@database/databaseConnection");
+const { knex } = dbConnection();
+
+/**
+ * Gets data for the Divisional Project Reports - Project Dashboard report.
+ *
+ * @param           requestParams portfolio: Optional list of portfolio_ids to limit report to. If empty, returns data for all portfolios.
```

Comment:
optionally, this docblock could be updated to describe the parameters that will be passed for this specific report type.

Reply:
I have added the on extra param in on line 7 comment.